backend/lib/seeds/test_survey_questions.csv

```diff
@@ -1,18 +1,27 @@
-15,"If yes, please confirm your name",1,""
-16,"If yes, please confirm your email address",2,""
-17,"If yes, please confirm your phone number",3,""
+1,"Are you the homeowner?",0,"Yes/No (can we leave this info with you for the homeowner?)"
+2,"Have you heard of heat pumps for home heating and cooling?",0,"Yes/No"
+3,"Here are some common benefits associated with heat pumps - are any of these important to you? Which is your top priority?",0,"Save money on energy bills/Having cooling as well as heating/Quieter than AC/Reducing use of fossil fuels & climate change/I like heat pumps more than my current HVAC system/If I like heat pumps more than my current HVAC system"
+4,"Have you or would you consider(ed) heat pumps for home heating? Or for cooling?",0,"Yes, for heating, not for cooling/Yes for cooling, not for heating/Yes, for both/No - heard of them but have not considered them for either/No answer - Not Available"
+5,"If you didn't consider heat pumps for either or both heating/cooling, why not?",0,"Didn't know they could do heating & cooling/Only need help with one: heating or cooling/Didn't think heat pumps could be efficient & cost effective doing heating & cooling/Didn't know how to get more info & begin the installation process/Too expensive/Home not compatible with heat pumps/Other"
+6,"If you did consider heat pumps for either or both heating & cooling, how far have you taken your interest in heat pumps?",0,"Basic research & Looked at a website/Decided on a plan/Contacted installers & Got quotes/Installed a heat pump/Other"
+7,"If you have considered but have not installed a heat pump yet, what factors have prevented you from moving forward? Was it...",0,"Didn't understand the options, details & next steps/Too expensive/Complexity, too much time, or hassle to do paperwork/Installation concerns & Bad installer experience/Home not compatible with heat pumps/Other"
+8,"Have you heard of Mass Save [wait for reaction]... the energy efficiency program?",0,"Yes/No"
+9,"If Yes, have you had contact?",0,"Yes/No"
+10,"If Yes to having contact, what was the extent of your contact?",0,"Talked on the phone/Scheduled a Mass Save visit/Had a Mass Save visit/Installed something in your home with Mass Save help/Other"
+11,"If answer to extent of contact is anything other than having a Mass Save visit then: What stopped you?",1,""
+12,"If answered NO to contact with Mass Save, then: have you heard of any of the following local partners of Mass Save:",0,"ABCD/RISE/Homeworks Energy/CET/ClearResult/None of the above"
+13,"If answered YES to contact with Mass Save, then what was the extent of your contact?",0,"Talked on the phone/Scheduled a Mass Save visit/Had a Mass Save visit/Installed something in your home with Mass Save help/Other/None of the above"
+14,"If you could save money on your utility bills with a state-discounted or free heating & cooling system upgrade, would you like to learn more?",0,"Yes/No"
```

Comment:
Line 12,13,14: Reworded these as google doc used numbered question references "If Yes to Question #"